test/sample-test/check_notebook_results.py

```diff
@@ -18,46 +18,49 @@
 
 ###### Input/Output Instruction ######
 # args:
-#   experiment: where the test run belong, only necessary when a job is submitted.
+#   experiment: where the test run belong, only necessary when a job is
+#               submitted.
 #   namespace: where the pipeline system is deployed.
 #   testname: test name in the json xml
 #   result: name of the file that stores the test result
 #   exit_code: the exit code of the bash command that runs the test.
 
+
 # Parsing the input arguments
 def parse_arguments():
   """Parse command line arguments."""
 
   parser = argparse.ArgumentParser()
-  parser.add_argument('--experiment',
-                      type=str,
-                      help='The experiment name')
-  parser.add_argument('--namespace',
-                      type=str,
-                      default='kubeflow',
-                      help="namespace of the deployed pipeline system. Default: kubeflow")
-  parser.add_argument('--testname',
-                      type=str,
-                      required=True,
-                      help="Test name")
-  parser.add_argument('--result',
-                      type=str,
-                      required=True,
-                      help='The path of the test result that will be exported.')
-  parser.add_argument('--exit-code',
-                      type=str,
-                      required=True,
-                      help='The exit code of the bash command that runs the test.')
+  parser.add_argument('--experiment', type=str, help='The experiment name')
```

Comment:
Are there any actual changes in this files?

We usually separate refactoring or restyling PRs from change PRs. This way the diff is much smaller and the reviewers can focus on the actual change.

Reply:
Thanks for your suggestion. Reverted this change.